src/components/Confirm/CaptureViewer/index.js

```diff
@@ -21,7 +20,7 @@ const CaptureViewer = ({
     blob={blob}
     id={id}
     isDocument={method === 'document'}
-    previewOrientation={previewOrientation}
+    isCroppedView={filename && filename.includes('document_capture.')}
```

Comment:
I think it's better not to rely on the filename for the UI logic. Can you maybe include an additional parameter to the capture to tell us if it's a live capture or not. And if it's a document live capture you display the cropped view

Reply:
Good point. I'll add it as a additional parameter to the capture since that can still get passed on to the Confirmation component.